### ISSUE TITLE
STATUS_FAIL_INTERNAL_ERROR: FATAL SIGNAL 11 in secp256k1 

### ISSUE THREAD

**Issue by fixone** [ElementsProject/lightning#1166]
While in process of updating lightningd to the latest version (as a result of closing #1072 ) noticed that the daemon was stopped and debug.log said
```
2018-03-03T23:01:20.402Z lightning_gossipd(1946): peer_in WIRE_PING
2018-03-03T23:01:20.402Z lightning_gossipd(1946): peer_out WIRE_PONG
2018-03-03T23:01:30.611Z lightning_gossipd(1946): peer_in WIRE_PING
2018-03-03T23:01:30.611Z lightning_gossipd(1946): peer_out WIRE_PONG
2018-03-03T23:01:31.110Z lightningd(1938): lightning_channeld-027c4d2f938f5ee5184fbf0c1b3152a806243b5ec1e53cff814f439d3c18ceceec chan #38: peer_in WIRE_PING
2018-03-03T23:01:31.110Z lightningd(1938): lightning_channeld-027c4d2f938f5ee5184fbf0c1b3152a806243b5ec1e53cff814f439d3c18ceceec chan #38: Got ping, sending WIRE_PONG
2018-03-03T23:01:31.110Z lightningd(1938): lightning_channeld-027c4d2f938f5ee5184fbf0c1b3152a806243b5ec1e53cff814f439d3c18ceceec chan #38: peer_out WIRE_PONG
2018-03-03T23:01:33.668Z lightning_gossipd(1946): peer_in WIRE_PING
2018-03-03T23:01:33.668Z lightning_gossipd(1946): peer_out WIRE_PONG
2018-03-03T23:01:38.706Z lightning_gossipd(1946): peer_in WIRE_PING
2018-03-03T23:01:38.706Z lightning_gossipd(1946): peer_out WIRE_PONG
2018-03-03T23:01:39.794Z lightning_gossipd(1946): Received channel_update for channel 1282461:6:0(1) now ACTIVE
2018-03-03T23:01:43.113Z lightning_gossipd(1946): Received channel_update for channel 1285557:17:0(1) now DISABLED
2018-03-03T23:01:45.792Z lightning_gossipd(1946): peer_in WIRE_PING
2018-03-03T23:01:45.792Z lightning_gossipd(1946): peer_out WIRE_PONG
2018-03-03T23:01:48.722Z lightning_gossipd(1946): backtrace: common/subdaemon.c:81 (subdaemon_setup) 0x41553e
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: (null):0 ((null)) 0x7f368be094af
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: src/field_5x52_impl.h:231 (secp256k1_fe_set_int) 0x44d8f2
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: src/group_impl.h:243 (secp256k1_gej_set_ge) 0x44d8f2
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: src/group_impl.h:419 (secp256k1_gej_add_ge_var) 0x44d8f2
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: src/ecdsa_impl.h:237 (secp256k1_ecdsa_sig_verify) 0x45750a
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: src/secp256k1.c:317 (secp256k1_ecdsa_verify) 0x45750a
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: bitcoin/signature.c:196 (IsValidSignatureEncoding) 0x41f0e4
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: gossipd/routing.c:588 (routing_channel_new) 0x40f6b0
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: gossipd/routing.c:987 (handle_channel_update) 0x41088c
2018-03-03T23:01:48.723Z lightning_gossipd(1946): backtrace: gossipd/gossip.c:508 (handle_pong) 0x403cfa
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: gossipd/gossip.c:840 (forget_peer) 0x404b9d
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: ccan/ccan/io/io.c:106 (io_new_conn_) 0x42e58c
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: ccan/ccan/io/io.c:443 (io_close) 0x42f063
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: ccan/ccan/io/io.c:454 (io_close_taken_fd) 0x42f0a1
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: ccan/ccan/io/fdpass/fdpass.c:41 (do_fd_recv) 0x430970
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: /usr/include/valgrind/valgrind.h:6778 (VALGRIND_PRINTF) 0x4085f9
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: (null):0 ((null)) 0x7f368bdf482f
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: (null):0 ((null)) 0x4029f8
2018-03-03T23:01:48.724Z lightning_gossipd(1946): backtrace: (null):0 ((null)) 0xffffffffffffffff
2018-03-03T23:01:48.724Z lightning_gossipd(1946): STATUS_FAIL_INTERNAL_ERROR: FATAL SIGNAL 11
```
The only problem that I have is that I did the pull & make before noticing the crash and I'm not sure at which commit the binary was built, can only say that the current version is "v0.5.2-2016-11-21-2206-g9b3e8f0"

**Comment by cdecker** [comment]
Marked as duplicate of #1072 since I think this should have been addressed by #1150, please update and let us know if it still appears. If yes I'll remove the duplicate flag :wink:

**Comment by cdecker** [comment]
Seems we can't reproduce this issue anymore, assuming it was fixed in #1150.
